### PR TITLE
Api change

### PR DESCRIPTION
**Pull request by mgautierfr**

### CODE REVIEW

**Comment by mgautierfr** [comment]
Need https://github.com/openzim/libzim/pull/277

File: src/zimrecreate.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ class Article : public zim::writer::Article         //Article class that will be
 
     bool shouldIndex() const
     {
-        return getMimeType().find("text/html") == 0;
+        return !isRedirect() && getMimeType().find("text/html") == 0;
```
Comment (kelson42) [inline src/zimrecreate.cpp]: I thought we have decided it was important to index redirects too!

Comment (mgautierfr) [reply src/zimrecreate.cpp]: We redirect the title all the time, whatever is `shouldIndex()`. `shouldIndex` is about full text indexing.

Saying that, I realize that we also index binary content (as image, and such) and we should not. But this is a issue to open on libzim side (I will do).

Comment (mgautierfr) [reply src/zimrecreate.cpp]: https://github.com/openzim/libzim/issues/278

Comment (mgautierfr) [reply src/zimrecreate.cpp]: I've removed this change.

**Pull request merged**